scripts/ops/users_github_id_unique_index.sql

```
@@ -0,0 +1,10 @@
+-- Required pre-deploy step: build the GitHub identity index without blocking writes.
```

SQLFluff (4.2.2)

[error] Line is too long (85 > 80).

(LT05)